solutions/src/test/java/ru/sberbank/school/task13/cacheproxy/CacheProxyTest.java

```diff
@@ -0,0 +1,79 @@
+package ru.sberbank.school.task13.cacheproxy;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import ru.sberbank.school.task13.cacheproxy.testclasses.TestInterface;
+import ru.sberbank.school.task13.cacheproxy.testclasses.TestInterfaceImpl;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class CacheProxyTest {
+    private static String pathStr;
+    private static TestInterface testInterface;
+    @BeforeAll
```

Comment:
&#39;METHOD_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-10527000>*